src/constants.js

```diff
@@ -1,45 +1,43 @@
-const constant = {
-  arr: [
-    "AC",
-    "+/-",
-    "%",
-    "÷",
-    "7",
-    "8",
-    "9",
-    "x",
-    "4",
-    "5",
-    "6",
-    "-",
-    "1",
-    "2",
-    "3",
-    "+",
-    "0",
-    ".",
-    "="
-  ],
-  colors: [
-    "#505050",
-    "#505050",
-    "#505050",
-    "#FF9500",
-    "#D4D4D2",
-    "#D4D4D2",
-    "#D4D4D2",
-    "#FF9500",
-    "#D4D4D2",
-    "#D4D4D2",
-    "#D4D4D2",
-    "#FF9500",
-    "#D4D4D2",
-    "#D4D4D2",
-    "#D4D4D2",
-    "#FF9500",
-    "#D4D4D2",
-    "#D4D4D2",
-    "#FF9500"
-  ]
-};
-export default constant;
+export const VALUES_ARRAY = [
+    'AC',
+    '+/-',
+    '%',
+    '÷',
+    '7',
+    '8',
+    '9',
+    'x',
+    '4',
+    '5',
+    '6',
+    '-',
+    '1',
+    '2',
+    '3',
+    '+',
+    '0',
+    '.',
+    '=',
+];
+export const COLORS = [
+    '#505050',
+    '#505050',
+    '#505050',
+    '#FF9500',
+    '#D4D4D2',
+    '#D4D4D2',
+    '#D4D4D2',
+    '#FF9500',
+    '#D4D4D2',
+    '#D4D4D2',
+    '#D4D4D2',
+    '#FF9500',
+    '#D4D4D2',
+    '#D4D4D2',
+    '#D4D4D2',
+    '#FF9500',
+    '#D4D4D2',
+    '#D4D4D2',
+    '#FF9500',
+];
+export const NUMBERS = [4, 5, 6, 8, 9, 10, 12, 13, 14, 16, 17];
```

Comment:
I don't know what these are used for and the variable name doesn't help me neither. Please give it a better name.